홍석민/week3/2023.03.13/boj11866/11866.py

```diff
@@ -0,0 +1,18 @@
+from collections import deque
+
+def solution(circle: deque, K: int):
+    counter = 0
+    while circle:
+        counter += 1
+        if counter % K == 0:
+            print(circle.popleft(), end='')
+            if circle:
+                print(', ', end='')
+        else:
+            circle.append(circle.popleft())
+
+N, K = map(int, input().split())
+circle = deque([i for i in range(1, N + 1)])
```

Comment:
```suggestion
circle = deque(i for i in range(1, N + 1))
```
굳이 리스트로 안묶어줘도 iter로 받아서 생성 가능해요!

Reply:
그런 방법이....